frontend/src/views/admin/InviteCodePanelView.vue

```diff
@@ -231,19 +181,19 @@ const generateInvites = async () => {
   pageError.value = '';
   pageNotification.value = '';
 
-  const response = await call(`invite/generate/${generateCodeAmount.value}`).post().json();
+  const response: InviteListResponse = await call(`invite/generate/${generateCodeAmount.value}`).post().json();
 
   const { data, error } = response;
 
   if (error.value) {
-    const errorObj = data.value?.detail;
+    const errorObj = (data.value as Exception)?.detail;
 
     if (!errorObj) {
       pageError.value = t('error.somethingWentWrong');
     } else if (errorObj instanceof Array) {
       pageError.value = errorObj.map((err) => err.msg).join('\n');
```

Comment:
@MelissaAutumn I couldn't figure out which case that is? When will `data.value` hold an array of error objects with `msg` property?

Reply:
I believe it's when there's multiple pydantic validation errors at once. I don't remember why this could come up.

Reply:
Ah I see, alright. Let's keep it that way for now.